From 1de16ce758d013375e0bc6386e6cc23780fc85b2 Mon Sep 17 00:00:00 2001
From: Kentaro Hayashi <kenhys@gmail.com>
Date: Sat, 4 May 2024 21:39:41 +0900
Subject: [PATCH 1/2] debian/rule: fix embedded path of installed path

Without this change, build root path will be
embedded into epics-base.pc.

See documentation/RELEASE_NOTES.md.

Signed-off-by: Kentaro Hayashi <kenhys@gmail.com>
---
 debian/rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index 12acc51..3f2716d 100755
--- a/debian/rules
+++ b/debian/rules
@@ -41,7 +41,7 @@ override_dh_auto_clean: debian/control
 	dh_auto_clean
 
 override_dh_auto_build:
-	$(MAKE) LINKER_USE_RPATH=NO
+	$(MAKE) LINKER_USE_RPATH=NO FINAL_LOCATION=/usr
 
 override_dh_auto_install:
 	$(MAKE) install
-- 
2.43.0

